core/plugins/org.polarsys.capella.core.platform.sirius.clipboard/src/org/polarsys/capella/core/platform/sirius/clipboard/actions/CapellaDiagramCopyAction.java

```diff
@@ -49,8 +49,12 @@ public void run(IAction action_p) {
     } else {
       // Copying graphical and semantic elements
       List<? extends EObject> toCopy = getCopyPasteSelection();
-      CapellaDiagramCopyCommand cmd = new CapellaDiagramCopyCommand(toCopy);
-      MiscUtil.transactionallyExecute(toCopy, cmd);
+      toCopy.removeIf(element -> (TitleBlockHelper.isTitleBlockAnnotation(LayerUtil.getSiriusElement((View)element).getTarget())));

```

Comment:
This code should be added directly in `org.polarsys.capella.core.platform.sirius.clipboard.actions.AbstractCopyPasteAction.getCopyPasteSelection()`.

Since both `CapellaDiagramCopyAction` and `CapellaDiagramPasteAction` inherit from that class, this avoids duplication. Furthermore this will solve the bug in `org.polarsys.capella.core.platform.sirius.clipboard.actions.CapellaDiagramCopyAction.isEnabled())` which wrongfully reports that a copy paste is possible only to be executed on 0 elements afterwards.